__test__/backend.test.js

```diff
@@ -204,34 +206,39 @@ it('should convert an invitation and user into a valid organization instance', a
 
 
 it('should create a test campaign', async () => {
-
-  const [user, invite] = await Promise.all([createUser(), createInvite()])
-  const organization = await createOrganization(user, "test org", user.id, invite.data.createInvite.id)
-
   const campaignTitle = "test campaign"
-  const campaign = await createCampaign(user, campaignTitle, "test description", organization.data.createOrganization.id)
-  expect (campaign.data.createCampaign.title).toBe(campaignTitle)
+  testCampaign = await createCampaign(testAdminUser, campaignTitle, "test description", testOrganization.data.createOrganization.id)
 
+  expect (testCampaign.data.createCampaign.title).toBe(campaignTitle)
 })
 
-
 it('should create campaign contacts', async () => {
-  const [user, invite] = await Promise.all([createUser(), createInvite()])
-  const organization = await createOrganization(user, "test org", user.id, invite.data.createInvite.id)
-
-  const campaign = await createCampaign(user, "test campaign", "test description", organization.data.createOrganization.id)
-
-  const contact = await createContact(campaign.data.createCampaign.id)
-
-  expect (contact.campaign_id).toBe(parseInt(campaign.data.createCampaign.id))
+  const contact = await createContact(testCampaign.data.createCampaign.id)
 
+  expect (contact.campaign_id).toBe(parseInt(testCampaign.data.createCampaign.id))
 })
 
-// it('should add texters to a organization', async () => {
-//
-//
-//
-// })
+it('should add texters to a organization', async () => {
+  testTexterUser = await createUser({
```

Comment:
We don't need to do this as part of this PR, but I'm tempted to suggest we move the `createUser()`, etc to a library file, and then also have a `createTexter()` which fills in arbitrary user data, so we can run it 100 times in one test easily. #future